curriculum/challenges/english/25-front-end-development/lab-markdown-to-html-converter/66f55eac933ff64ce654ca74.md

```diff
@@ -1054,11 +1080,24 @@ const convertMarkdown = () => {
     return markdown;
 }
 
-markdownInput.addEventListener('input', () => {
+markdownInput.addEventListener("input", () => {
     const html = convertMarkdown();
     const out = document.getElementById('html-output');
     const preview = document.getElementById('preview');
     out.textContent = html;
     preview.innerHTML = html;
 });
+
+markdownInput.addEventListener("keyup",() => {
+    let debounceTimeout;
+    clearTimeout(debounceTimeout);
+    debounceTimeout = setTimeout(() => {
+    const html = convertMarkdown();
+    const out = document.getElementById('html-output');
+    const preview = document.getElementById('preview');
+    out.textContent = html;
+    preview.innerHTML = html;
+  }, 300);
+})
+
```

Comment:
I'm not sure if second handler is needed. Changed tests are triggering two events to allow more flexibility in the solutions, but they don't need to handle both of them.